### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by Zweistein2**
minor improvements:

- Added method in AssetManager to return all loaded assets of a type
- Added method in AssetManager to return the amount of all assets ever queued (both loaded and not loaded)
- Added method in AssetManager to return the amount of all assets already loaded
- Added method in AssetManager to return the amount of all assets currently in queue
- Added method for setting the loglevel directly on Mdx.log

### CODE REVIEW

File: core/src/main/java/org/mini2Dx/core/assets/AssetManager.java [hunk 1]
```diff
@@ -196,7 +210,35 @@ public float getProgress() {
...
+	 */
+	public float getQueuedAssets() {
+		return queuedAssets;
+	}
+
+	/**
+	 * Returns the number of already loaded assets
+	 * @return A value more than or equal zero
+	 */
+	public float getCompletedTasks() {
```
Comment (peanutbother) [inline core/src/main/java/org/mini2Dx/core/assets/AssetManager.java:225]: why do we need floats here?
there shouldn't be e.g. 1.5 tasks completed

File: core/src/main/java/org/mini2Dx/core/assets/AssetManager.java [hunk 2]
```diff
@@ -196,7 +210,35 @@ public float getProgress() {
 		return completedTasks / queuedAssets;
 	}
 
+	/**
+	 * Returns the number of all assets (both loaded and not loaded) ever queued
+	 * @return A value more than or equal zero
+	 */
+	public float getQueuedAssets() {
```
Comment (peanutbother) [inline core/src/main/java/org/mini2Dx/core/assets/AssetManager.java:217]: why do we need floats here?
there shouldn't be e.g. 1.5 tasks queued

File: core/src/main/java/org/mini2Dx/core/assets/AssetManager.java [hunk 3]
```diff
@@ -196,7 +210,35 @@ public float getProgress() {
...
+	 */
+	public float getCompletedTasks() {
+		return completedTasks;
+	}
+
+	/**
+	 * Returns the number of currently queued assets
+	 * @return A value more than or equal zero
+	 */
+	public float getUnfinishedTasks() {
```
Comment (peanutbother) [inline core/src/main/java/org/mini2Dx/core/assets/AssetManager.java:233]: why do we need floats here?
there shouldn't be e.g. 1.5 tasks unfinished

Comment (tomcashman) [reply core/src/main/java/org/mini2Dx/core/assets/AssetManager.java:233]: There could be 1.5 tasks finished as a task might be 50% complete if it runs over several frames or asynchronously.

Comment (peanutbother) [reply core/src/main/java/org/mini2Dx/core/assets/AssetManager.java:233]: I see what you mean, but currently there are only whole-number increments of this number

**Pull request merged**